_dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue

```diff
@@ -46,26 +46,68 @@
             class="spinner"
           />
         </div>
-        <b-form-row
-          class="mt-2"
+        <b-container
           v-else
+          class=" maxw-sm-800"
         >
+          <div class="d-flex justify-content-around">
+            <div class="text-center mb-2">
+              <img
+                class="rounded-circle mb-1"
+                src="@/assets/images/google-icon-grey.svg"
+                width="20"
+                height="20"
+              >
+              <p>
+                {{ $t('productFeedSettings.summary.tableHeader1') }}
+              </p>
+            </div>
+            <div :style="cssProps" />
```

Comment:
If all you're doing is having a background image, why not adding the background image to the parent ? l53.